README.md

```diff
@@ -10,24 +10,47 @@ A JavaScript plugin to warn users about links to private pages. Places a :lock:
 
 At 18F, this is used on public sites that contain links to internal content like private GitHub repositories or Google Docs. Rather than write two versions to redact those links, this allows us to publish new content and give a warning to both staff and external readers.
 
-## Usage
+## Installation
 
 Compatible with modern browsers (IE 9+). No dependencies.
 
+### Script
+
+Private Eye can be included as a normal script on your page, exposing a `PrivateEye` global.
+
 ```html
 <script src="private-eye.js"></script>
```

Comment:
Good idea! Let's take care of that separately: https://github.com/18F/private-eye/issues/10